crates/uv-workspace/src/build_backend.rs

```diff
@@ -0,0 +1,168 @@
+    #[option(
+        default = r#"[]"#,
+        value_type = "list[str]",
+        example = r#"
+            source-exclude = ["*.bin"]
+        "#
+    )]
+    pub source_exclude: Vec<String>,
+
+    /// Glob expressions for files and directories to exclude from the wheel.
+    #[option(
+        default = r#"[]"#,
+        value_type = "list[str]",
+        example = r#"
+            wheel-exclude = ["*.bin"]
+        "#
+    )]
+    pub wheel_exclude: Vec<String>,
+
+    /// Data includes for wheels.
+    ///
+    /// Each entry is a directory, whose contents are copied to the matching directory in the wheel
+    /// in `<name>-<version>.data/(purelib|platlib|headers|scripts|data)`. Upon installation, this
+    /// data is moved to its target location, as defined by
+    /// <https://docs.python.org/3.12/library/sysconfig.html#installation-paths>. Usually, small
+    /// data files are included by placing them in the Python module instead of using data includes.
+    ///
+    /// - `scripts`: Installed to the directory for executables, `<venv>/bin` on Unix or
+    ///   `<venv>\Scripts` on Windows. This directory is added to `PATH` when the virtual
+    ///   environment  is activated or when using `uv run`, so this data type can be used to install
+    ///   additional binaries. Consider using `project.scripts` instead for Python entrypoints.
+    /// - `data`: Installed over the virtualenv environment root.
+    ///
+    ///     Warning: This may override existing files!
+    ///
+    /// - `headers`: Installed to the include directory. Compilers building Python packages
+    ///   with this package as build requirement use the include directory to find additional header
+    ///   files.
+    /// - `purelib` and `platlib`: Installed to the `site-packages` directory. It is not recommended
+    ///   to uses these two options.
+    // TODO(konsti): We should show a flat example instead.
+    // ```toml
+    // [tool.uv.build-backend.data]
+    // headers = "include/headers",
+    // scripts = "bin"
+    // ```
+    #[option(
+        default = r#"{}"#,
+        value_type = "dict[str, str]",
+        example = r#"
+            data = { "headers": "include/headers", "scipts": "bin" }
+        "#
+    )]
+    pub data: WheelDataIncludes,
+}
+
+impl Default for BuildBackendSettings {
+    fn default() -> Self {
+        Self {
+            module_root: PathBuf::from("src"),
+            module_name: None,
+            source_include: Vec::new(),
+            default_excludes: true,
+            source_exclude: Vec::new(),
+            wheel_exclude: Vec::new(),
+            data: WheelDataIncludes::default(),
+        }
+    }
+}
+
+/// Data includes for wheels.
+///
+/// See `BuildBackendSettings::data`.
+#[derive(Default, Deserialize, Serialize, OptionsMetadata, Debug, Clone, PartialEq, Eq)]
+// `deny_unknown_fields` to catch typos such as `header` vs `headers`.
+#[serde(default, rename_all = "kebab-case", deny_unknown_fields)]
```

Comment:
NB this is a future-proofing hazard for schema users (if the schema is ever out of date it will freak out about the field being missing -- I think that's fine here, especially if we make the schema in schemastore hotlinked to our github repo).

Reply:
I'm making a bet here that wheels won't get another data field (the list has been the same since 2012) and it's more likely that users mistype one of the data names (that's why it's the only type with `deny_unknown_fields` in the build backend).